test_framework/scripts/longhorn-setup.sh

```diff
@@ -56,9 +56,12 @@ install_cluster_autoscaler(){
 
 install_csi_snapshotter_crds(){
     CSI_SNAPSHOTTER_REPO_URL="https://github.com/kubernetes-csi/external-snapshotter.git"
-    CSI_SNAPSHOTTER_REPO_BRANCH="v6.2.1"
     CSI_SNAPSHOTTER_REPO_DIR="${TMPDIR}/k8s-csi-external-snapshotter"
 
+    wget "https://raw.githubusercontent.com/longhorn/longhorn/${LONGHORN_REPO_BRANCH}/deploy/longhorn-images.txt" -O "/tmp/longhorn-images.txt"
```

Comment:
Is there any specific reason we're not using `LONGHORN_REPO_URI`? Without it, what happens when we are using a remote repository and a branch that isn't part of `longhorn/longhorn`? There should be a way to convert it to a raw URL.

Reply:
Refined. Thank you!